cocotb/share/makefiles/simulators/Makefile.ius

```diff
@@ -71,8 +71,9 @@ endif
 # which does not play nice with our multi language, so we supply GPI_EXTRA=cocotbvhpi if needed.
 
 GPI_EXTRA:=
+GPI_ARGS = -loadvpi $(LIB_DIR)/libcocotbvpi:vlog_startup_routines_bootstrap
```

Comment:
Do you know why we suddenly need the bootstrapping?

Reply:
No

Reply:
Before it was somehow automatic because of name libvpi.

Reply:
For me,  it's working without the explicit name of the startup routine in both Xcelium 19 (modified and used both ``Makefile.ius`` and ``Makefile.xcelium``) and Incisive 15 (used ``Makefile.ius``).

Reply:
Xcelium/IUS will used default vlog_startup_routines symbol only if vpi library name is `libvpi.so`